tools/make/helm.mk

```diff
@@ -1,35 +1,44 @@
 # This is a wrapper to manage helm chart
 #
-# All make targets related to helmß are defined in this file.
+# All make targets related to helms are defined in this file.
 
 include tools/make/env.mk
 
 IMAGE_PULL_POLICY ?= IfNotPresent
 OCI_REGISTRY ?= oci://docker.io/envoyproxy
 CHART_NAME ?= gateway-helm
+ADDONS_CHART_NAME ?= gateway-addons-helm
```

Comment:
can we follow similar pattern as we use for `image` ?
https://github.com/envoyproxy/gateway/blob/51a3a9dd6e71daa568db066b34cd62eae9c69185/tools/make/image.mk#L11
this will allow us to build and package individual package using commands like `make helm-package.gateway-helm` and also use top level commands like `make helm-package` to run for all charts

Reply:
done